packages/bot-web-ui/src/pages/bot-builder/quick-strategy/quick-strategy.tsx

```diff
@@ -42,6 +43,9 @@ const getErrorMessage = (dir: 'MIN' | 'MAX', value: number, type = 'DEFAULT') =>
 };
 
 const FormikWrapper: React.FC<TFormikWrapper> = observer(({ children }) => {
+    const {
+        client: { currency },
+    } = useStore();
```

Comment:
These could break if the client is not present in the store. Let's not destructure.